canonical-websites/www.ubuntu.com
# Update terms boxes

## Done

Add new content to legal page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal>
- Check that page matches the [copy doc](https://docs.google.com/document/d/19yTLYWu7rEErPrO0xDsqMeN8tcWKBjHuarw0tjxRlhY/edit)


## Issue / Card

Fixes #2824
